src/datasets/ga-dashboard.dataset.ts

```diff
@@ -0,0 +1,131 @@
+import * as random from 'random';
+import { TimeSeriesPoint } from './metas/types';
+import { DAY } from '../utils/timeUnits';
+import { Dataset } from './types';
+import { PreferenceMeta } from '../services/preference/types';
+import { createLineChartMeta } from './metas/line-chart.meta';
+import { createBarChartMeta, createPieChartMeta } from './metas/categorical.meta';
+import { TimeSeriesQueryOptions } from './queries/time-series.query';
+import { CategoricalQueryOptions } from './queries/categorical.query';
+import { SUMMARIZATION } from '../services/summarization/types';
+
+export type Config = {};
+
+export const configMeta: PreferenceMeta<Config> = {};
+
+export function create(config: Config): Dataset {
+  const sessionsByCountryPoints = [
+    { x: 'US', y: 4820 },
+    { x: 'India', y: 2330 },
+    { x: 'Canada', y: 790 },
+    { x: 'UK', y: 340 },
+    { x: 'Japan', y: 270 },
+  ];
+  const sessionsByDevicePoints = [
+    { x: 'Desktop', y: 6860 },
+    { x: 'Mobile', y: 2990 },
+    { x: 'Tablet', y: 730 },
+    { x: 'Others', y: 150 },
+  ];
+
+
+  const activeUsersPointsLength = 28;
+  const activeUsersPoints: TimeSeriesPoint[] = [];
+
+  const activeUsersWeekPattern = [-40, 60, 70, 80, 90, 100, -30];
+  const activeUsersOffset = 100;
+  const activeUsersGrowingRate = 10;
+
+  const startDate = new Date(Date.now() - activeUsersPointsLength * DAY);
+  const rand = random.normal(0, 10);
+
+  for (let i = 1; i <= activeUsersPointsLength; i++) {
+    const x = new Date(startDate.getTime() + i * DAY);
+    const y = activeUsersOffset + i * activeUsersGrowingRate + activeUsersWeekPattern[x.getDay()] + rand();
+    activeUsersPoints.push({ x, y });
+  }
+
+  const sessionsByCountrySummarizationMetas = [

```

Comment:
nit: would be good to test if the metas created have the correct summarizations

Reply:
should be covered by the unit tests of `SummarizationControlServices`